packages/ui-extensions/src/surfaces/admin/components/Modal/Modal.ab.doc.ts

```diff
@@ -0,0 +1,40 @@
+import {AdminReferenceEntityTemplateSchema} from '../../docs-types';
+import sharedContent from '../../../../docs/shared/components/Modal';
+
+const data: AdminReferenceEntityTemplateSchema = {
+  ...sharedContent,
+  thumbnail: '/assets/templated-apis-screenshots/admin/components/modal.png',
+  isVisualComponent: true,
+  definitions: [
+    {
+      title: 'Properties',
+      description: '',
+      type: 'Modal',
+    },
+    {
+      title: 'Events',
+      description:
+        'Learn more about [registering events](/docs/api/app-home/using-polaris-components#event-handling).',
+      type: 'ModalEvents',
+    },
+    {
+      title: 'Slots',
+      description: '',
+      type: 'ModalSlots',
+    },
+  ],
+  defaultExample: {
+    codeblock: {
+      title: 'Code',
+      tabs: [
+        {
+          code: './examples/default.html',
+          language: 'preview',
+          customStyles: 'min-height: 300px;',
```

Comment:
How do you pass in multiple styles here? Should this be an array?

Reply:
It is just a string. Inline styles like you'd add to a style attribute (e.g. `style="color: red; background: blue;"`)

Reply:
I've refined this to support a CSS Style Object. Types included